src/shared/components/instances.tsx

```diff
@@ -170,6 +170,12 @@ class InstanceCard extends Component<InstanceCardProps, InstanceCardState> {
             >
               {i18n.t("explore")}
             </a>
+            <a
+              className="btn btn-secondary btn-outline text-white max-md:btn-block normal-case"
```

Comment:
I think `max-md:btn-block` should be `md:btn-block`. At least, I don't believe tailwind has a `max-md` modifier.

Also, is the `normal-case` necessary? I would think that the text would already be in normal case.

Reply:
Its for targeting widths smaller and including md: https://tailwindcss.com/docs/responsive-design#targeting-a-breakpoint-range.  So it'll show the block-width button only on smaller devices.

Normal case can probably be removed, I'll abstract this since its the same as the other button classes anyway.